src/utils/store/globalStore.ts

```diff
@@ -0,0 +1,88 @@
+import { create } from 'zustand';
```

Comment:
jeśli ten global store będzie Ci siępowiekszał to późnie warto go podzielić na mniejsze obiekty. Ale to tak żebyś na przyszłość miałą na uwadzę :)